roles/remove_cloudnativepg_sandbox/tasks/cnpg-sandbox-monitored-cluster.yml

```diff
@@ -0,0 +1,18 @@
+# Original work Copyright 2019-2020, EnterpriseDB Corporation
```

Comment:
This file does not look like a task file. Maybe we can move/transform this content into variables defined in `defaults/main.yml`.